systemtests/tests/python-bareos/testrunner

```diff
@@ -18,8 +18,8 @@ JobName=backup-bareos-fd
 
 
 # Directory to backup.
-# This directory will be created by setup_data "$@"().
-BackupDirectory="${tmp}/data"
+# This directory will be created by setup_data().
+export BackupDirectory="${tmp}/data"
```

Comment:
Is this not already done now in environment Line 21?

Reply:
Rmeoved line.